gitzones/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM alpine:latest
+
+RUN apk add --no-cache openssh git python3
+
+RUN adduser -D -s /usr/bin/git-shell git \
+  && mkdir /home/git/.ssh \
+  && ln -s /etc/ssh/keys/git_authorized_keys /home/git/.ssh/authorized_keys \
+  && passwd -u git
```

Comment:
The unlocks a locked password. Why is that needed here?

Reply:
Otherwise SSH refuses login

Reply:
That's weird. On my machines, I always lock my root accounts using `passwd -l root`, and can still do key-based login ...

Reply:
I confirmed that password-based login is impossible for user git even after passwd -u